occult/lib/all.dart

```diff
@@ -443,20 +443,23 @@ Future<void> createFirebaseRC(String projectname) async {
 }
 
 Future<void> enableGcloudArtifactRegistry(String project) async {
+  final String gcloudCommand = Platform.isWindows ? 'gcloud.cmd' : 'gcloud';
+
```

Comment:
Dont do this, do this in the process calling utilities, dont change commands or create variables for each command. Future commands now have to follow this undocumented pattern. The process utility should handle this automatically. Abstract it.